Makefile

```diff
@@ -95,6 +95,9 @@ network:
 	cd integration/network/macvlan && \
 	bats macvlan_driver.bats
 
+ramdisk:
```

Comment:
Silly question - are we actually running the test? Does it need to be added to the `UNION` at the top of the file so the `test:` rule invokes it for instance?